src/base.jl

```diff
@@ -561,6 +564,28 @@ function get_contributing_device_mapping(sys::System)
     return services
 end
 
+"""
+    get_buses(sys::System, aggregator::AggregationTopology)
```

Comment:
This is nice. The next step for users will be to collect the other components in an AggregationTopology. That doesn't seem straightforward to me. Something like:
```julia
[c for c in get_components(Generator, sys) if get_bus(c) in get_buses(sys, area)]
```
It gets more complicated for Branches. 
I'm wondering if we should add a method for this, or if you think it's reasonable for users to figure it out?

Reply:
We need to implement this only for injection devices at minimum

Reply:
Added for StaticInjection devices.  If we need to do something similar for DynamicInjection devices then we should add an issue to track it.